connect/connect-c8/element-template-c8/src/main/java/io/miragon/miranum/connect/adapter/in/c8/elementtemplates/schema/Binding.java

```diff
@@ -0,0 +1,291 @@
+package io.miragon.miranum.connect.adapter.in.c8.elementtemplates.schema;
+
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * property binding
+ * <p>
+ * Specifying how the property is mapped to BPMN or Zeebe extension elements and attributes
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "type",
+        "name",
+        "source",
+        "key"
+})
+
+public class Binding
+{
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    @JsonPropertyDescription("The type of a property binding")
+    private Binding.Type type;
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    @JsonPropertyDescription("The name of a property binding")
+    private String name;
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    @JsonPropertyDescription("The source value of a property binding (zeebe:output)")
+    private String source;
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    @JsonPropertyDescription("The key value of a property binding (zeebe:taskHeader)")
+    private String key;
+
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    public Binding.Type getType()
+    {
+        return type;
+    }
+
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    public void setType(Binding.Type type)
+    {
+        this.type = type;
+    }
+
+    public Binding withType(Binding.Type type)
+    {
+        this.type = type;
+        return this;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public String getName()
+    {
+        return name;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public void setName(String name)
+    {
+        this.name = name;
+    }
+
+    public Binding withName(String name)
+    {
+        this.name = name;
+        return this;
+    }
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
```

Comment:
Are the javadocs generated? They do not seem to be correct

Reply:
No they are not generated. But the text is from this [JSON](https://unpkg.com/@camunda/zeebe-element-templates-json-schema@0.8.0/resources/schema.json). Or what do you mean by incorrect?

Reply:
Ah ok, the value in brackets is probably just an example. It seemed to me that this is the only value this binding type can hold.